src/pages/userInvitation/UserInvitationPageStore.js

```diff
@@ -300,7 +301,20 @@ export const useUserInvitationPageStore = defineComponentStore(
 		}
 
 		const {redirectToPage} = useUrl('management/settings/access');
-		const isSubmitting = ref(false);
+		const isSubmitting = ref(invitationPayload.value.disabled ? true : false);
```

Comment:
```suggestion
		const isSubmitting = ref(!!invitationPayload.value.disabled);
```
or if value is always a boolean for `invitationPayload.value.disabled`, we can just reference it directly:
```suggestion
		const isSubmitting = ref(invitationPayload.value.disabled);
```

Reply:
Yes I saw this one